array-methods/7-array/index.html

```diff
@@ -12,35 +12,90 @@
       // Go through todo.md before start solving.
       function countAllPeople() {
         // your code goes here
-        console.log(got);
+        
+        totatlPeople = got.houses.reduce( (acc, cv) => {
+          acc +=  cv.people.length;
+          return acc;
+        }, 0);
+
+        console.log(totatlPeople);
```

Comment:
Return `totatlPeople` instead of `console.log` because that will give you more control of what you want to do with that.